src/main/java/mate/academy/EventManager.java

```diff
@@ -1,19 +1,28 @@
 package mate.academy;
 
+import java.util.concurrent.ConcurrentLinkedQueue;
+
 public class EventManager {
-    public void registerListener(EventListener listener) {
+    private ConcurrentLinkedQueue<EventListener> queueu;
```

Comment:
```suggestion
    private ConcurrentLinkedQueue<EventListener> queue;
```
but let's not use such a general name. Consider giving a more meaningful name